packages/koa-performance/src/enums.ts

```diff
@@ -1,6 +1,7 @@
 export enum LifecycleMetric {
   TimeToFirstByte = 'time_to_first_byte',
   TimeToFirstContentfulPaint = 'time_to_first_contentful_paint',
+  TimeToLargestContentfulPaint = 'time_to_largest_contentful_paint',
```

Comment:
Not shipped yet so still possible to change this, but while I agree that lcp/fcp would be better, it'd be a breaking change to rename ttfcp -> fcp at this point, and I'd prefer to avoid them if possible since it'd break any existing dashboards using that metric. I think given the choice, it's better to match the existing standards.

The good news is, that we're looking at moving to OpenTelemetry and a new library that's being built will consume these but emit it's own metrics and since we're "starting fresh" there we can use the new names for the OTel metrics